### PR TITLE
fix \ character in errMsg

### PR DESCRIPTION
**Pull request by dsonet**
\ is escape character in JavaScript.

### CODE REVIEW

File: lib/tracer.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ module.exports = class Tracer {
         err.toString()
           .replace(/[\r\n]+/g, '<br/>')
           .replace(/"/g, '\\"')
+          .replace(/\\/g, '\\\\')
```
Comment (lizheming) [inline lib/tracer.js]: L75 has replaced to `\"`, then you replace `\` to `\\`, does it has some problem?

Comment (dsonet) [inline lib/tracer.js]: oh... ya, then should change the order.

Comment (lizheming) [inline lib/tracer.js]: OK, I changed it~

**Comment by lizheming** [comment]
Thank you for your pr, new version 1.0.7 has published

**Pull request merged**